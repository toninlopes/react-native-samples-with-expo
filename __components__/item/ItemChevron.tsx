import React from 'react';
import { View, StyleSheet } from 'react-native';
import { Ionicons } from '@expo/vector-icons';

const ItemChevron: React.FC = () => {
  return (
    <View style={styles.view}>
      <Ionicons name='chevron-forward-outline' size={28}/>
    </View>
  );
}

const styles = StyleSheet.create({
  view: {
    // paddingHorizontal: 12,
  },
});

export default ItemChevron;