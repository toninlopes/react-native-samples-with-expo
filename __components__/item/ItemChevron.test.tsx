import React from 'react';
import renderer from 'react-test-renderer';
import ItemChevron from './ItemChevron';

describe('ItemChevron', () => {
  test('When it is rendered', () => {
    const itemChevron = renderer.create(<ItemChevron />).toJSON();
    expect(itemChevron).toMatchSnapshot();
  });
});