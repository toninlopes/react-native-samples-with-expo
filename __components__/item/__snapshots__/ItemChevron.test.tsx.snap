// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemChevron When it is rendered 1`] = `
<View
  style={{}}
>
  <Text />
</View>
`;
